From 5e52d778894c2490b17cdef72d3bb0909ba574a2 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Thu, 10 May 2018 10:11:47 +0800
Subject: [PATCH] ASoC: rockchip: multicodecs: fix the missing return 0 at
 hw_param

Here the -ENOTSUPP is not an error if there is no set_sysclk
implement in codec_dai and cpu_dai, we need to return 0 that
it's correct.

Change-Id: I969e3eacba39ac8d6c94ddcc27c60cc110d66156
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 sound/soc/rockchip/rockchip_multicodecs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/rockchip/rockchip_multicodecs.c b/sound/soc/rockchip/rockchip_multicodecs.c
index 8a6de82fc888..8b74dfdc0bff 100644
--- a/sound/soc/rockchip/rockchip_multicodecs.c
+++ b/sound/soc/rockchip/rockchip_multicodecs.c
@@ -60,6 +60,8 @@ static int rk_multicodecs_hw_params(struct snd_pcm_substream *substream,
 		goto out;
 	}
 
+	return 0;
+
 out:
 	return ret;
 }
-- 
2.35.3

